src/components/Nav/HelpDropdown.tsx

```diff
@@ -3,37 +3,45 @@ import { Dropdown, Icon, MenuItem } from 'patternfly-react';
 
 import AboutUIModal from '../About/AboutUIModal';
 import { Component } from '../../store/Store';
+import DebugInformationContainer from '../../containers/DebugInformationContainer';
 
 type HelpDropdownProps = {
   status: { [key: string]: string };
   components: Component[];
   warningMessages: string[];
 };
-type HelpDropdownState = {};
 
-class HelpDropdown extends React.Component<HelpDropdownProps, HelpDropdownState> {
-  about: any;
+class HelpDropdown extends React.Component<HelpDropdownProps> {
+  about: React.RefObject<AboutUIModal>;
+  debugInformation: React.RefObject<any>;
 
   constructor(props: HelpDropdownProps) {
     super(props);
+    this.about = React.createRef<AboutUIModal>();
+    this.debugInformation = React.createRef();
   }
 
+  openAbout = () => {
+    this.about.current!.open();
+  };
+
+  openDebugInformation = () => {
+    // Using wrapped component, so we have to get the wrappedInstance
+    this.debugInformation.current!.getWrappedInstance().open();
+  };
+
   render() {
     return (
       <>
-        <AboutUIModal
-          ref={about => {
-            this.about = about;
-          }}
-          status={this.props.status}
-          components={this.props.components}
-        />
+        <AboutUIModal ref={this.about} status={this.props.status} components={this.props.components} />
+        <DebugInformationContainer ref={this.debugInformation} />
         <Dropdown componentClass="li" id="help">
           <Dropdown.Toggle useAnchor={true} className="nav-item-iconic">
             <Icon type="pf" name="help" />
           </Dropdown.Toggle>
           <Dropdown.Menu>
-            <MenuItem onClick={() => this.about.open()}>About</MenuItem>
+            <MenuItem onClick={this.openDebugInformation}>Send debug information</MenuItem>
```

Comment:
Menu Items should start uppercase like: `Show Debug Information`

Reply:
Ok